tg_bot.py

```diff
@@ -20,19 +21,23 @@ class Telegram_Bot(object):
     def __init__(self, dirpath):
         self.dirpath = dirpath
 
+    def tweepy_auth(self):
+        api = Twitter_Bot.nawab_twitter_authenticate()
```

Comment:
'nawab_twitter_authenticate()' does the tweepy authentication for you, Why wrapping it up again in a function?